submissions/Levi123/domjs/script.js

```diff
@@ -0,0 +1,69 @@
+    },
+    {
+        id:4,
+        series: '4 series',
+        images: 'https://www.bmw.ua/content/bmw/marketB4R1/bmw_ua/uk_UA/all-models/4-series/coupe/2020/bmw-4-series-coupe-highlights/jcr:content/par/mosaicgallery/items/mosaicgalleryitem_2045026459/image.transform/mosaic485/image.1597922414251.jpg',
+        information: 'Безкомпромісний нонконформіст: абсолютно новий BMW 4 серії Coupé рішуче та елегантно виходить за рамки всіх сучасних правил. Незалежний дизайн і класичний силует купе підкреслюють його прогресивний характер. Спортивна легкість плавно переходить на дорогу, де вражаюча маневреність і привабливі характеристики сприяють підвищенню рівня адреналіну.',
+    },
+    {
+        id:5,
+        series: '5 series',
+        images: 'https://www.bmw.ua/content/bmw/marketB4R1/bmw_ua/uk_UA/all-models/5-series/sedan/2020/bmw-5-series-sedan-highlights/jcr:content/par/mosaicgallery_d34b/items/mosaicgalleryitem_97/image.transform/mosaic963/image.1597323989179.jpg',
+        information: 'BMW 5 серії Седан — втілення спортивного бізнес-седана. З першого погляду він демонструє стильний атлетизм, який особливо виражається у динамічному силуеті та лаконічному дизайні. Прості поверхні та чіткі контури надають елегантному зовнішньому вигляду сучасну та технологічну атмосферу. Елегантний і функціональний інтерєр доповнює інноваційні амбіції автомобіля технологіями майбутнього та спортивними особливостями. зкомпромісний нонконформіст: абсолютно новий BMW 4 серії Coupé рішуче та елегантно виходить за рамки всіх сучасних правил. Незалежний дизайн і класичний силует купе підкреслюють його прогресивний характер. Спортивна легкість плавно переходить на дорогу, де вражаюча маневреність і привабливі характеристики сприяють підвищенню рівня адреналіну.',
+    },
+]
+
+const buttons = document.querySelectorAll('.button');
+// const button = document.querySelector('.about-product__navigation')
+const mainInfoBlock = document.querySelector('.about-product__info-block');
+const mainTitle = document.createElement('h1');
+const mainImage = document.createElement('img');
+const mainText = document.createElement('p');
+
+function generateInformationAboutModel(event){
+    const buttonId = Number(event.target.dataset.id);
+    informationAboutModel.forEach((value) => {
+        if (value.id === buttonId){
+            mainInfoBlock.prepend(mainTitle);
+            mainTitle.classList.add('about-product__main-title');
+            mainTitle.innerHTML = `About BMW ${value.series}`;
+
+            mainInfoBlock.append(mainText);
+            mainText.classList.add('about-product__description');
+            mainText.innerHTML = value.information;
+
+            mainInfoBlock.append(mainImage);
+            mainImage.classList.add('about-product__main-image');
+            mainImage.src = value.images;
+        }
+    })
+}
+
+buttons.forEach((button) => {
+    button.addEventListener('click', generateInformationAboutModel);
```

Comment:
You must add an event listener only for the menu list.

Reply:
> You must add an event listener only for the menu list.

but I seem to have done so?

Reply:
> You must add an event listener only for the menu list.

i have button class only on buttons and use it in this method

Reply:
find me in telegram madmaxWMFU

Reply:
[Read this article](https://davidwalsh.name/event-delegate#:~:text=Event%20delegation%20allows%20you%20to,just%20how%20event%20delegation%20works.) hope it helps)